unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/WearableController.cs

```diff
@@ -202,4 +202,18 @@ public bool IsLoadedForBodyShape(string bodyShapeId)
 
         return wearable.data.representations.FirstOrDefault(x => x.bodyShapes.Contains(bodyShapeId))?.mainFile == lastMainFileLoaded;
     }
+
+    public void SetSSAOEnabled(bool ssaoEnabled)
```

Comment:
Please remember to reset this value to `true` when the controller is cleaned up or we may get pooled wearables with this badly toggled.